src/content/developers/docs/smart-contracts/languages/index.md

```diff
@@ -105,11 +106,15 @@ For more information, [read the Vyper rationale](https://vyper.readthedocs.io/en
 
 - [Documentation](https://vyper.readthedocs.io)
 - [Vyper by Example](https://vyper.readthedocs.io/en/latest/vyper-by-example.html)
+- [More Vyper by Example](https://vyper-by-example.org/)
 - [GitHub](https://github.com/vyperlang/vyper)
-- [Vyper Gitter Chatroom](https://gitter.im/vyperlang/community)
+- [Vyper commmunity Discord chat](https://discord.gg/SdvKC79cJk)
 - [Cheat Sheet](https://reference.auditless.com/cheatsheet)
+- [Smart contract development frameworks and tools for Vyper](https://ethereum.org/en/developers/docs/programming-languages/python/)
```

Comment:
We don't need the whole URL for internal links FYI we can drop everything up to the lang path.

```suggestion
- [Smart contract development frameworks and tools for Vyper](/developers/docs/programming-languages/python/)
```